examples/Microsoft.Spark.CSharp.Examples/MachineLearning/Sentiment/README.md

```diff
@@ -0,0 +1,193 @@
+
+## Solution
+
+We'll first train an ML model using ML.NET, and then we'll create a new application that uses both .NET for Apache Spark and ML.NET.
+
+> **Note:** All of the necessary files (trained ML model, .NET for Spark + ML.NET application code, training and testing datasets) have been included in this project. You can follow the steps below to understand how the project/files were developed, recreate them yourself, and then adapt the steps to future applications.
+
+## ML.NET
+
+[ML.NET](dot.net/ml) is an open source and cross-platform machine learning framework that allows .NET developers to easily integrate ML into .NET apps without any prior ML experience. 
+
+### 1. Download Model Builder
+
+We'll use ML.NET to build and train a model through [Model Builder](https://dotnet.microsoft.com/apps/machinelearning-ai/ml-dotnet/model-builder), a Visual Studio extension that provides an easy to understand visual interface to build, train, and deploy machine learning models. Model Builder can be downloaded [here](https://marketplace.visualstudio.com/items?itemName=MLNET.07). 
+
+![Model Builder](https://mlnet.gallerycdn.vsassets.io/extensions/mlnet/07/16.0.1909.2101/1569301315962/add-machine-learning.gif)
+
+**Note:** You can also develop a model without Model Builder. Model Builder just provides an easier way to develop a model that doesn't require prior ML experience. 
+
+### 2. Build and Train Your Model
+
+Follow the [Model Builder Getting Started Guide](https://dotnet.microsoft.com/learn/machinelearning-ai/ml-dotnet-get-started-tutorial/intro) to train your model using the sentiment analysis scenario. 
+
+Follow the steps to:
+
+* Create a new C# Console App
+* Pick the **Sentiment Analysis** scenario
+* Train using the **Amazon.csv** dataset
+
+![Sentiment Analysis Model Builder](https://dotnet.microsoft.com/static/images/model-builder-vs.png?v=9On8qwmGIXdAyX_-zAmATwYU7fd7tzem-_ojnv1G7XI)
+
+### 3. Generate Model and Code
+
+In the last step of using Model Builder, you'll produce a zip file containing the trained ML.NET model. In the image below, you can see it's contained in **MLModel.zip.**
+
+![ML.NET Zip and Files](https://github.com/bamurtaugh/spark/blob/SparkMLNet/examples/Microsoft.Spark.CSharp.Examples/MachineLearning/images/modelbuilder5proj.PNG)
+
+You'll also generate C# code you can use to consume your model in other .NET apps, like the Spark app we'll be creating.
+
+![Generated Code](https://github.com/bamurtaugh/spark/blob/SparkMLNet/examples/Microsoft.Spark.CSharp.Examples/MachineLearning/images/modelbuilder5code.PNG)
+
+### 4. Add ML.NET to .NET for Apache Spark App
+
+You have a few options to start creating a .NET for Apache Spark app that uses this ML.NET code and trained model. Make sure that in any app you develop, you've downloaded the [Microsoft.ML NuGet Package](https://www.nuget.org/packages/Microsoft.ML).
```

Comment:
```suggestion
You have a few options to start creating a .NET for Apache Spark app that uses this ML.NET code and trained model. Make sure that in any app you develop, you've added the [Microsoft.ML.FastTree NuGet Package](https://www.nuget.org/packages/Microsoft.ML.FastTree).
```

Reply:
Do you know why we need FastTree?   Does that mean the `Next Steps` in the screenshot in section 3 is incorrect ?

Reply:
So I just retrained the ML model with Model Builder (since I needed to train it with the new column names), and the recommended algorithm changed. I'm thinking that depending upon which algorithm is chosen and thus used within the trained model, that affects which nugets the user needs to download.

I referenced the new MLModel.zip I generated, and tried removing the reference to `Microsoft.ML.FastTree` and just including `Microsoft.ML` instead, and it worked. So I think it depends on the model.

I'll add an explanation about this to the readme.

Reply:
Added an explanation, let me know what you think.

Reply:
Sounds good!

Reply:
Great, thanks! Marking this as resolved then (but let me know if you have any other feedback about it).